element-hq/element-x-android
# Read user avatar from cache

## Type of change


## Content


Ensure the user avatar in the room list is rendered even if there is no network. The avatar will also be rendered faster in case of slow network, since it's now read from cache.

In this PR the user profile will be read from cache, previously 2 network requests was trigger on in received Push. This will speed up the display of the notification and reduce the network request.

## Motivation and context


Fast UI

Closes #2488

## Screenshots / GIFs



## Tests



- Login to an account with a user which has an avatar: the avatar is rendered in the room list at top start.
- Kill the app
- enter airplane mode, so the device has no way to reach the homeserver
- start the app
- observe that the user avatar is rendered.

Previously the avatar was not rendered in this case.

## Tested devices

- OS version(s):

## Checklist
